responsibleai/responsibleai/managers/data_balance_manager.py

```diff
@@ -0,0 +1,433 @@
+        self._pos_label = pos_label
+        self._custom_data = custom_data
+        self._backend = backend
+
+    def compute(self):
+        if not self._validate():
+            return
+
+        self._data_balance_measures = (
+            DataBalanceManager._get_data_balance_measures(
+                df=self._df,
+                cols_of_interest=self._cols_of_interest,
+                target_column=self._target_column,
+                pos_label=self._pos_label,
+                backend=self._backend,
+            )
+        )
+
+    def get(self):
+        # TODO docs and implementation
+        ...
+
+    def list(self):
+        # TODO docs and implementation
+        ...
+
+    def get_data(self):
+        return (
+            self._data_balance_measures if self._data_balance_measures else {}
+        )
+
+    @property
+    def name(self):
+        return ManagerNames.DATA_BALANCE
+
+    def _save(self, path):
+        """Abstract method to save the manager.
+
+        :param path: The directory path to save the manager to.
+        :type path: str
+        """
+        # TODO docs and implementation
+        ...
+
+    @staticmethod
+    def _load(path, rai_insights):
+        """Static method to load the manager.
+
+        :param path: The directory path to load the manager from.
+        :type path: str
+        :param rai_insights: The loaded parent RAIInsights.
+        :type rai_insights: RAIInsights
+        :return: The BaseManager after loading.
+        :rtype: BaseManager
+        """
+        # TODO docs and implementation
+        ...
+
+    def _validate(self) -> bool:
+        if self._custom_data is not None:
+            self._df = self._custom_data
+        elif (
+            self._train is not None
+            and not self._train.empty
+            and self._test is not None
+            and not self._test.empty
+        ):
+            self._df = pd.concat([self._train, self._test])
+        elif self._train is not None and not self._train.empty:
+            self._df = self._train
+        elif self._test is not None and not self._test.empty:
+            self._df = self._test
+        else:
+            warnings.warn(
+                (
+                    "Either `train`, `test`, or `custom_data` must be provided"
+                    " to compute data balance measures."
+                )
+            )
+            return False
+
+        if not all([self._cols_of_interest, self._target_column]):
+            warnings.warn(
+                (
+                    "Both `cols_of_interest` and `target_column` must be"
+                    " provided to compute data balance measures."
+                )
+            )
+            return False
+
+        if self._backend not in SUPPORTED_BACKENDS:
+            warnings.warn(
+                (
+                    "`backend` must be one of the supported backends:"
+                    f" {SUPPORTED_BACKENDS} to compute data balance measures."
```

Comment:
I'm not sure if this is printed in the desired format?
```{SUPPORTED_BACKENDS}```
I think you actually need to do a join on this, otherwise it looks a bit weird to the user?
```
>>> list = ["a", "b", "c"]
>>> print(list)
['a', 'b', 'c']
>>> print(f" {list}")
 ['a', 'b', 'c']
>>> ", ".join(list)
'a, b, c'
```